includes/commenting-block-functions.php

```diff
@@ -85,30 +85,25 @@ function gc_reassigning_deleted_user( $id, $reassign ) {
 		LIKE %s",
 		$like
 	) );
-	$current_user_id = get_current_user_id();
+	$user_to_reassign = $reassign ? $reassign : get_current_user_id();
```

Comment:
This can be also `$user_to_reassign = $reassign ?: get_current_user_id();` but I wasn't sure what is the lowest PHP version this plugin supports.